pkg/utils/etcdutil/etcdutil_test.go

```diff
@@ -366,7 +365,8 @@ func ioCopy(ctx context.Context, dst io.Writer, src io.Reader, enableDiscard *at
 			return nil
 		default:
 			if enableDiscard.Load() {
-				io.Copy(io.Discard, src)
+				_, err := io.Copy(io.Discard, src)
```

Comment:
why return here?

Reply:
Does the src discard? Do we still need to do the following things?

Reply:
It seems to return directly without  `io.Copy`?